packages/repl/src/lib/workers/bundler/index.js

```diff
@@ -1,4 +1,5 @@
 import { rollup } from '@rollup/browser';
+import { exports as exprotsResolver, legacy as legacyResolver } from 'resolve.exports';
```

Comment:
typo in "exports" and should probably use `_` rather than camelCase as it's an internal variable
```suggestion
import { exports as exports_resolver, legacy as legacy_resolver } from 'resolve.exports';
```